cmd/state-svc/manager.go

```diff
@@ -75,6 +75,7 @@ func (s *serviceManager) Stop() error {
 
 	ctx, cancel := context.WithTimeout(context.Background(), svcmanager.MinimalTimeout)
 	defer cancel()
+	svcmgr.SetCheckVersion(false)
```

Comment:
I'd rather see this as internal logic of the svcmgr. ie. the manager should know not to care about version checking for the "stop" event.